docs/content/preview/explore/multi-region-deployments/asynchronous-replication-ysql.md

```diff
@@ -16,165 +16,115 @@ menu:
-   👕 Claim your free t-shirt at https://www.yugabyte.com/community-rewards/
-   ```
-   
-1. Create and start your second local cluster that simulates "Data Center = West" by running the following `yugabyted start` command from your YugabyteDB home directory:
-
-     ```sh
-   ./bin/yugabyted start --base_dir=datacenter-west --listen=127.0.0.2
-   ```
-   
-    The preceding command starts a one-node cluster using IP address of `127.0.0.2` and creates `datacenter-west` as the base directory. Expect to see an output similar to the following:
-
-   
-
-     ```output
-   Starting yugabyted...
-   ✅ System checks
-   
-   +--------------------------------------------------------------------------------------------------+
-   |                                            yugabyted                                             |
-   +--------------------------------------------------------------------------------------------------+
-   | Status              : Running. Leader Master is present                                          |
-   | Web console         : http://127.0.0.2:7000                                                      |
-   | JDBC                : jdbc:postgresql://127.0.0.2:5433/yugabyte?user=yugabyte&password=yugabyte  |
-   | YSQL                : bin/ysqlsh -h 127.0.0.2  -U yugabyte -d yugabyte                           |
-   | YCQL                : bin/ycqlsh 127.0.0.2 9042 -u cassandra                                     |
-   | Data Dir            : /Users/myuser/yugabyte-2.7.2.0/datacenter-west/data                        |
-   | Log Dir             : /Users/myuser/yugabyte-2.7.2.0/datacenter-west/logs                        |
-   | Universe UUID       : ad78f70c-0741-4c7e-b610-315d55d7f248                                       |
-   +--------------------------------------------------------------------------------------------------+
-   🚀 yugabyted started successfully! To load a sample dataset, try 'yugabyted demo'.
-   🎉 Join us on Slack at https://www.yugabyte.com/slack
-   👕 Claim your free t-shirt at https://www.yugabyte.com/community-rewards/
-     ```
+Create the two data centers as follows:
 
+1. Create and start the local cluster that simulates "Data Center - East" by running the following command from your YugabyteDB home directory:
 
-## Create tables
+    ```sh
+    ./bin/yugabyted start \
+                    --advertise_address=127.0.0.1 \
+                    --base_dir=/tmp/ybd1
+    ```
+
+    This starts a one-node local cluster using the IP address of `127.0.0.1` and creates `datacenter-east` as the base directory.
+
+1. Create and start the second local cluster that simulates "Data Center - West" by running the following command:
+
+    ```sh
+    ./bin/yugabyted start \
+                    --advertise_address=127.0.0.2 \
+                    --base_dir=/tmp/ybd2
+    ```
+
+    This starts a one-node cluster using IP address of `127.0.0.2` and creates `datacenter-west` as the base directory.
```

Comment:
You mentioned `--base_dir=/tmp/ybd1` as the base directory. This should probably be `--base_dir=/tmp/datacenter-west` or something similar.